runhouse/servers/servlet.py

```diff
@@ -570,9 +594,19 @@ def add_secrets(self, message: Message):
                 output_type=OutputType.EXCEPTION,
             )
 
-    def call(self, module, method=None, args=None, kwargs=None, serialization="json"):
+    def call(
+        self,
+        module,
+        method=None,
+        args=None,
+        kwargs=None,
+        serialization="json",
+        token=None,
+    ):
         self.register_activity()
         module = obj_store.get(module)
+        self.validate_resource_access(module, token)
```

Comment:
I think it's fine to do this here for now, but most likely I'd move it into the obj_store's `get` method later. Also, it'd probably be cleaner for most of the auth logic to just live in the object store, rather than split it between the obj store and servlet.